### ISSUE TITLE
Moving of fluorescence polarization

### ISSUE THREAD

**Issue by djc183** [BioAssayOntology/BAO#33]
The term "Fluorescence Polarization" (currently assay method component --> physical detection method --> fluorescence method, http://www.bioassayontology.org/bao#BAO_0000003) would fit better in the tree under immunoassay-->ex situ immunoassay (http://www.bioassayontology.org/bao#BAO_0000466) and renamed to "Fluorescence Polarization Assay".

While fluorescence polarization is technically a process/method that is exploited in the assay, it seems to be very specific to a subtype of immunoassay, similarly to ELISA, rather than a more generalized reaction or process.

**Comment by sschurer** [comment]
are you sure about that. binding , but not immuno.

**Comment by djc183** [comment]
Yes, it is used with other interaction types, but FPIA (fluorescence polarization immunoassay) is a common implementation of the polarization concept.

We can say that the design method is polarization, and have the individual assays, such as FIPA, under their respective sections as methods (i.e. ex situ immunoassay mentioned above).  Would that be reasonable?

**Comment by sschurer** [comment]
makes sense. Thank you

**Comment by handemcginty** [comment]
fluorescence polarization stays as is, FPIA (fluorescence polarization immunoassay) will be added under immunoassay-->ex situ immunoassay
